vignettes/approved.Rmd

```diff
@@ -16,45 +16,51 @@ knitr::opts_chunk$set(
 library(logrx)
 ```
 
-## 1. Create a named list
-#### The named list contains the functions approved for use for each package. If all functions for a package are approved for use, list "All".
+# 1. Create a named list
+The named list contains the functions approved for use for each package. If all 
+functions for a package are approved for use, list "All".
 
 ```{r}
 approved_pkgs <- list(base = "mean",
                       dplyr = "All")
 approved_pkgs
 ```
 
-## 2. Build `approved.rds`
-#### Pass the named list through `build_approved()` to build your tibble.
-#### If We create a temp directory to save this for illustration.
+# 2. Build `approved.rds`
+Pass the named list through `build_approved()` to build your tibble. We create 
+a temp directory to save this for illustration.
 
 ```{r}
 build_approved(approved_pkgs)
 ```
 
-## 3. Save as `approved.rds`
-####  You can use the `file` argument in `build_approved()` to save approved.rds instead of return the tibble.
+# 3. Save as `approved.rds`
+You can use the `file` argument in `build_approved()` to save approved.rds 
+instead of return the tibble.
 
 ```{r}
 dir <- tempdir()
 
 build_approved(approved_pkgs, file = file.path(dir, "approved.rds"))
 ```
 
-## 4. Update the `logrx.approved` option
-### Update the `logrx.approved` option to point to your `approved.rds` location. We recommend setting this in your `.Rprofile`.
+# 4. Update the `logrx.approved` option
+Update the `logrx.approved` option to point to your `approved.rds` location. We 
+recommend setting this in your `.Rprofile`.
 
 ```{r}
 options(log.rx.approved = file.path(dir, "approved.rds"))
 ```
 
-## 5. You're done. Let's axecute! 
-#### `logrx` will take it from there. When each program is executed, packages and functions will be compared to `approved.rds` and if any unapproved use is found, it will be logged within the "Unapproved Package and Functions" section.
+# 5. You're done. Let's axecute! 
+`logrx` will take it from there. When each program is executed, packages and 
+functions will be compared to `approved.rds` and if any unapproved use is 
+found, it will be logged within the "Unapproved Package and Functions" section.
 
-## Example
+# Example
 
-Let's write a simple script summarizing mean mpg from mtcars. We save this as `mpg.R` in the temporary directory `dir` and `axecute()` it.
+Let's write a simple script summarizing mean mpg from mtcars. We save this as `
+mpg.R` in the temporary directory `dir` and `axecute()` it.
 
 ```{r}
 library(dplyr)
```

Comment:
can you insert message = FALSE so the dplyr stuff about masking does not appear.

Reply:
https://reprex.tidyverse.org/articles/suppress-startup-messages.html there's a function solely for this thankfully